pkg/operator/controller/controller.go

```diff
@@ -283,15 +293,36 @@ func (r *reconciler) ensureDNSDeleted(dns *operatorv1.DNS) error {
 // ensureDNSNamespace ensures all the necessary scaffolding exists for
 // dns generally, including a namespace and all RBAC setup.
 func (r *reconciler) ensureDNSNamespace() error {
-	ns := manifests.DNSNamespace()
-	if err := r.client.Get(context.TODO(), types.NamespacedName{Name: ns.Name}, ns); err != nil {
+	existingNamespace := corev1.Namespace{}
+	desiredNamespace := manifests.DNSNamespace()
+	if err := r.client.Get(context.TODO(), types.NamespacedName{Name: desiredNamespace.Name}, &existingNamespace); err != nil {
```

Comment:
Might be worth adding a function to `names.go` that would return the `types.NamespacedName` for the namespace.

Reply:
Do you want me to update it everywhere, or just in what I added/changed?

Reply:
Just in the code you're adding or changing.